Crash on 4.3 class cast exception

Phone: Samsung S3
OS: 4.3
CalendarView version: com.prolificinteractive:material-calendarview:1.0.1

Hi all

Hard to reproduce, but possible related to saving to bundle state. Here is the log:

```10-09 08:53:08.109 10931-10931/? E/AndroidRuntime: FATAL EXCEPTION: main
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.example/com.example.activities.NavigationActivity}: java.lang.ClassCastException: android.os.Parcelable[] cannot be cast to com.prolificinteractive.materialcalendarview.CalendarDay[]
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2295)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2349)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread.access$700(ActivityThread.java:159)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1316)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Handler.dispatchMessage(Handler.java:99)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Looper.loop(Looper.java:176)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread.main(ActivityThread.java:5419)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at java.lang.reflect.Method.invokeNative(Native Method)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at java.lang.reflect.Method.invoke(Method.java:525)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1046)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:862)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at dalvik.system.NativeStart.main(Native Method)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:  Caused by: java.lang.ClassCastException: android.os.Parcelable[] cannot be cast to com.prolificinteractive.materialcalendarview.CalendarDay[]
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.prolificinteractive.materialcalendarview.MaterialCalendarView$SavedState.<init>(MaterialCalendarView.java:988)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.prolificinteractive.materialcalendarview.MaterialCalendarView$SavedState.<init>(MaterialCalendarView.java:934)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.prolificinteractive.materialcalendarview.MaterialCalendarView$SavedState$1.createFromParcel(MaterialCalendarView.java:971)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.prolificinteractive.materialcalendarview.MaterialCalendarView$SavedState$1.createFromParcel(MaterialCalendarView.java:969)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Parcel.readParcelable(Parcel.java:2062)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Parcel.readValue(Parcel.java:1971)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Parcel.readSparseArrayInternal(Parcel.java:2284)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Parcel.readSparseArray(Parcel.java:1693)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Parcel.readValue(Parcel.java:2028)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Parcel.readMapInternal(Parcel.java:2255)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Bundle.unparcel(Bundle.java:223)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Bundle.getSparseParcelableArray(Bundle.java:1237)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:840)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1075)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1057)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.FragmentManagerImpl.dispatchCreate(FragmentManager.java:1855)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.Activity.onCreate(Activity.java:935)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.support.v4.app.BaseFragmentActivityDonut.onCreate(BaseFragmentActivityDonut.java:39)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.support.v4.app.FragmentActivity.onCreate(FragmentActivity.java:242)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.support.v7.app.AppCompatActivity.onCreate(AppCompatActivity.java:58)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.example.activities.TrackingActivity.onCreate(TrackingActivity.java:24)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.example.activities.BaseActivity.onCreate(BaseActivity.java:42)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.example.activities.NavigationActivity.onCreate(NavigationActivity.java:47)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.Activity.performCreate(Activity.java:5372)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1104)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2257)
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2349) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread.access$700(ActivityThread.java:159) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1316) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Handler.dispatchMessage(Handler.java:99) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.os.Looper.loop(Looper.java:176) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at android.app.ActivityThread.main(ActivityThread.java:5419) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at java.lang.reflect.Method.invokeNative(Native Method) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at java.lang.reflect.Method.invoke(Method.java:525) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1046) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:862) 
10-09 08:53:08.109 10931-10931/? E/AndroidRuntime:     at dalvik.system.NativeStart.main(Native Method) ```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,7 +1,7 @@
 Change Log
 ==========
 
-Version 1.1.0 *(2015-10-14)*
+Version 1.1.0 *(2015-10-19)*
 ----------------------------
 
 * New: Ability to disable month swiping with `setPagingEnabled()`

--- a/sample/build.gradle
+++ b/sample/build.gradle
@@ -17,7 +17,7 @@ dependencies {
     // You should use the commented out line below in you're application.
     // We depend on the source directly here so that development is easier.
     compile project(':library')
-    //compile 'com.prolificinteractive:material-calendarview:1.0.0'
+    //compile 'com.prolificinteractive:material-calendarview:1.1.0'
 
     compile 'com.android.support:appcompat-v7:23.1.0'
     compile 'com.android.support:recyclerview-v7:23.1.0'

